public/app/features/alerting/unified/components/receivers/ReceiversSection.tsx

```diff
@@ -28,14 +28,14 @@ export const ReceiversSection = ({
   addButtonTo,
   children,
   showButton = true,
-  exportLink,
   canReadSecrets = false,
+  showExport = false
 }: React.PropsWithChildren<Props>) => {
   const styles = useStyles2(getStyles);
-  const showMore = Boolean(exportLink);
+  const showMore = showExport;
```

Comment:
Does it make sense to have the `showMore` variable, given it has the same value as `showExport`?

Reply:
💯